internal/prototest/examplepb/example.pb.yarpc.go

```diff
@@ -740,7 +740,7 @@ var (
 
 // TestMessageNameParityYARPCClient is the YARPC client-side interface for the TestMessageNameParity service.
 type TestMessageNameParityYARPCClient interface {
-	MessageName(context.Context, *Get2NdMessageRequest, ...yarpc.CallOption) (*Get2NdMessageResponse, error)
+	MessageName(context.Context, *Get2ndMessageRequest, ...yarpc.CallOption) (*Get2ndMessageResponse, error)
```

Comment:
Why do we have changes in the proto files where only thrift generated file should have been touched? Are you up to date with dev? I think this change was introduced with https://github.com/yarpc/yarpc-go/pull/2078

Reply:
https://github.com/yarpc/yarpc-go/commits/wit/thriftrw_bump-and-gen seems to contain #2078.

I've rebased and ran `make generate SUPPRESS_DOCKER=1` again and it appears to still have this change.

Reply:
Hum interesting, @gandhikrishna do you know what is going on?

Reply:
So I pulled the branch `wit/thriftrw_bump-and-gen`, rebased with dev, and executed `make generate SUPPRESS_DOCKER=1`. Turns out it generates the correct goType (`Get2NdMessageRequest`). @witriew can you try this once again.

Reply:
Resolved this with @gandhikrishna and @Dogild.  Seems I needed to explicitly purge my version of `protoc-gen-yarpc-go`.